homeassistant/components/tesla.py

```diff
@@ -0,0 +1,108 @@
+"""
+Support for Tesla cars.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/vera/
+"""
+from collections import defaultdict
+
+import voluptuous as vol
+
+from homeassistant.util import slugify
```

Comment:
Please sort the import lines and imported names alphabetically, :abcd:, within each group builtin, 3rd party, home assistant.